chsdi/models/clientdata_dynamodb.py

```diff
@@ -73,7 +73,6 @@ def get(self):
 
 
 dynamodb_connection = DynamodbConnection()
-s3_connection = S3Connection()
```

Comment:
We shared connections accross requests with this approach (at least, that' what I assume).

Now we don't anymore. What is the impact of that?

Might we share connections accross requests for each profile?